packages/next/next-server/server/config.ts

```diff
@@ -181,6 +181,12 @@ export default function loadConfig(
       phase,
       userConfigModule.default || userConfigModule
     )
+    if (Object.keys(userConfig).length === 0) {
```

Comment:
I don't think we should be checking for the number of keys in the object. Shouldn't we be checking for `undefined` (or a loose `== null`)?

Reply:
In my testing, when I did

```js
// some-config.js

{ someProp: true }
```

```js
const config = require('./some-config.js') 
// config = {}
```

`config` was `{}`, not a falsy output.

Reply:
now that I think of it, technically this is a valid `next.config.js` file...

```js
module.exports = {}
```

so maybe there is another check I could do to see if the file has any exports defined 🤔

Reply:
I haven't been able to find a way to determine if a Node file actually contains a `module.exports` definition (short of reading the file via `fs.readFile` and parsing the string file contents). 

There is a way to get the Node `Module` object of the file (via `module.children.find`), but the contents are still `{}` 😢 

Any thoughts? I know that `module.exports = {}` is technically a valid `next.config.js` file, so checking via `Object.keys` isn't going to work under that assumption.